hw/ip/keymgr/doc/_index.md

```diff
@@ -250,6 +251,26 @@ See the tables below for an enumeration.
 *  During `Invalid` and `Disabled` states, the relevant key / outputs are updated regardless of the error.
 *  Only the relevant collateral is updated -> ie, advance / disable command leads to working key update, and generate command leads to software or sideload key update.
 
+## DICE Support
```

Comment:
@weicaiyang let me know if this is clear to you.,
I also realized i forgot to update the block diagram, i'll make sure to do that.

Reply:
Hi Tim, do we have a place to show what exact data these CDI are derived from (similar to [this](https://docs.opentitan.org/doc/security/specs/identities_and_root_keys/)), so that I can check them.

Reply:
Thanks Tim. The document update is clear to me.